include/flatbuffers/reflection.h

```diff
@@ -95,6 +95,13 @@ inline const Table *GetAnyRoot(const uint8_t *flatbuf) {
   return GetRoot<Table>(flatbuf);
 }
 
+inline Table *GetAnySizePrefixedRoot(uint8_t *flatbuf) {
+  return GetMutableSizePrefixedRoot<Table>(flatbuf);
+}
```

Comment:
nit: spaces between definitions.

Reply:
Ok, I will change this and for GetAnyRoot, which i copied.